torch/_dynamo/variables/builtin.py

```diff
@@ -1431,6 +1431,12 @@ def _unimplemented():
         if isinstance(left, TensorVariable):
             from .builder import wrap_fx_proxy_cls
 
+            if op is operator.is_ and isinstance(right, TensorVariable):
+                return ConstantVariable.create(
+                    id(left.as_proxy().node.meta["example_value"])
```

Comment:
Let's add a helper function that gets the `example_value` and errors with unimplemented if it doesn't exist.